sqldelight-studio-plugin/testData/completion/src/main/sqldelight/com/sample/ResultColumn.sq

```diff
@@ -0,0 +1,3 @@
+result_column
```

Comment:
ha interesting. Cool that the autocomplete still works. Thats what I would want.